sycl/doc/extensions/experimental/sycl_ext_oneapi_invoke_simd.asciidoc

```diff
@@ -227,9 +227,10 @@ in the `args` parameter pack according to the following rules:
   `operator T()` would be undefined, the value of the scalar variable passed
   to the SIMD function is undefined.
 
-- Arguments of type `std::tuple` are mapped as if each member of the tuple
-  was mapped individually (e.g. a `std::tuple<float, int>` becomes a
-  `std::tuple<sycl::ext::oneapi::experimental::simd<float, N>, sycl::ext::oneapi::experimental::simd<int, N>>`.
+- Arguments of type `sycl::ext::oneapi::experimental::flatten<std::array<T, M>>`
+  are converted to type `sycl::ext::oneapi::experimental::simd<T, N*M>`,
+  where `N` is the sub-group size of the calling kernel. The m-th element from each
+  work-item is stored consecutive.  
```

Comment:
I found the last sentence a little hard to understand.  Personally, I think something like this would be clearer:

> The first `simd` element is the first array element of the first work-item, followed by the first array element of the second work-item, etc.  The `simd` element `N` holds the second array element of the first work-item, followed by the second array element of the second work-item, etc.